### ISSUE TITLE
Interstitial ad show on start application and can not show multiple times

### ISSUE THREAD

**Issue by ildar-gilmanov** [kafeg/adctl#4]
I call adCtl.showAdMobInterstitial() on button clicked handler and remove all other calls to show intersitial ad window, but interstitial add showed on start and only on start application.

**Comment by adderly** [comment]
Yeah this will be fixed soon, the reloading of the Interstitial is/was not implemented.
